engine/order_manager.go

```diff
@@ -73,6 +73,7 @@ func (m *OrderManager) Start() error {
 	log.Debugln(log.OrderMgr, "Order manager starting...")
 	m.shutdown = make(chan struct{})
 	m.orderStore.wg.Add(1)
+	m.processOrders()
```

Comment:
After our discussions, I do not mind the use of the `time.After`, however, the thing I'm concerned with here is that `m.ProcessOrders()` is not called via a go routine. In likely the majority of cases, this shouldn't be an issue, but if you have performed many orders for many currencies for many exchanges, this lack of `go` call blocks the application from closing and from cancelling orders on closure. If a user is trading frequently, the impacts of such limitations are large.

See the following: 
```go
        tt := time.Now()
	m.processOrders()
	fmt.Println(time.Since(tt))
```
Populating a few exchange API keys of my own, even with quite limited trading, I got `3.569271958s`. However, when FTX was alive, this function for me would take > 10 seconds and I would not consider myself a super-active trader.

I am concerned about how it can block versus the recommendation I gave in the previous review. 

An alternative could also be to do this before `m.orderStore.wg.Add(1)`, so then its not blocking the closure